src/main/java/com/emc/mongoose/base/metrics/context/MetricsContextBase.java

```diff
@@ -59,18 +51,28 @@ public final long startTimeStamp() {
 	}
 
 	@Override
-	public final String id() {
-		return id;
+	public final Map metaData() {
+		return metaData;
+	}
+
+	@Override
+	public final String loadStepId() {
```

Comment:
there's no more need in methods like loadStepId, runId, opType, etc
because there's a method returning a metadata instance

Reply:
but it will make the code less readable